build-plugin/src/main/kotlin/thunderbird.library.android.gradle.kts

```diff
@@ -0,0 +1,35 @@
+plugins {
+    id("com.android.library")
+    id("org.jetbrains.kotlin.android")
+}
+
+android {
+    configureSharedConfig()
+
+    defaultConfig {
+        @Suppress("DEPRECATION")
+        targetSdk = ThunderbirdProjectConfig.androidSdkTarget
```

Comment:
Libraries probably shouldn't set `targetSdk`.

Reply:
Right, I had to set it as otherwise the `RelativeDateTimeFormatterTest` started to fail.

Reply:
I see. Please add a comment stating that we set `targetSdk` so Robolectric tests pick up on it. Maybe in the future we can directly configure Robolectric and get rid of this.

Reply:
This issue is related: [[Target SDK handling changes break robolectric tests [238666871] - Visible to Public - Issue Tracker](https://issuetracker.google.com/issues/238666871)](https://issuetracker.google.com/issues/238666871)